apps/ecommerce/frontend/src/components/Config/ConfigPage.tsx

```diff
@@ -132,6 +132,7 @@ const ConfigPage = () => {
                       type={def.type === 'Symbol' ? 'text' : 'number'}
                       maxLength={255}
                       isRequired={def.required}
+                      placeholder={def.placeholder}
```

Comment:
@whitelisab FYI i added `placeholder` support to the config / type for these to help with the UI/UX for "shop name"

https://github.com/contentful/apps/assets/492573/113a79fc-e4d6-480b-9ed4-7820fac4ffc9